.wiki/Rest-api.md

```diff
@@ -0,0 +1,46 @@
+# Artipie management Rest API
+
+Artipie provides Rest API to manage [repositories](./Configuration-Repository), [users](./Configuration-Credentials) 
+and [storages aliases](./Configuration-Storage#Storage-Aliases). API is self-documented with [Swagger](https://swagger.io/)
+interface, Swagger documentation pages are available on URLs `http://localhost:8081/api/index-{layout}.html` where
+`layout` is the layout you run Artipie with, `org` or `flat`.
+
+In Swagger documentation have three definitions - Repositories, Users and Auth Token. You can switch
+between the definitions with the help of "Select a definition" listbox.
+
+<img src="https://user-images.githubusercontent.com/14931449/193015387-3e25f937-7f23-4b27-884c-f183ca9dc8a0.png" alt="Swagger documentation" width="400"/>
+
+All Rest API endpoints require JWT authentication token to be passed in `Authentification` header. 
+The token can be issued with the help of `POST /api/v1/oauth/token` request on "Auth Token" 
+definition page in Swagger. Once token is received, copy it, open another definition, press 
```

Comment:
Is there a similar description in the Swagger documentation?
I think about a link on this doc.

Reply:
I do not think there is such documentation in swagger